chalicelib/lib/slack.py

```diff
@@ -144,13 +147,24 @@ def slack_payload_extractor(req):
         return "failed extracting payload", 200
 
 
-def verify_token(token):
-    if token == os.getenv('slack_token'):
+def verify_token(request, token):
+    #https://api.slack.com/docs/verifying-requests-from-slack
+
+    request_timestamp = request.headers['X-Slack-Request-Timestamp']
+
+    # create basestring out of v0 + timestamp + req_body
+    request_basestring = bytes('v0:' + request_timestamp + ':' + request, 'utf-8')
```

Comment:
Can't this be with nice f-strings instead? (i.e `f"v0:{request_timestamp}:{request}"`)

Reply:
good point!